src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,7 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          => // TODO do nothing; this needs to be checked in the preconditions
```

Comment:
As the comment suggests, this is something that needs to be checked in the preconditions, so should not be happening *here*. That's why I choose to keep it empty here and leave a TODO for future @jo-pol or future me to implement that precondition together with multiple others. The main point of this PR is to fix a non-exhaustive pattern match that would cause errors if bad input was given...